path_publisher:
  max_dis_btw_points : 0.5 # meters
  distance_to_slowdown_on_ends : 3
  path_resolution : 0.1 # 10 cms
  steering_limits_to_slow_down : 10 # if steering angle is more than this value, its associated speed reduces.

patrol:
  path_topic           : "odom_path"
  cmd_topic            : "/pure_pursuit/cmd_drive"
  pilot_cmd_in         : "/vehicle/cmd_drive_safe"
  wait_time_on_mission_complete  : 20  # secs
  google_map_center_always : true
  max_forward_speed : 2.5
  min_forward_speed : 1.5
  max_backward_speed : -1.2
  min_backward_speed : -0.3

pure_pursuit:
  min_look_ahead_dis : 3 # meters
  max_look_ahead_dis : 6 # meters
  speed_to_lookahead_ratio : 1
  speed_curvature_ratio : 2

stanley:
  control_gain : 0.1 # meters
  softening_gain : 1 # meters

carla_sim:
  speed : 0.3

